main.go

```diff
@@ -132,10 +141,32 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *graphiteAddr != nil {
+		go graphite.Graphite(metrics.DefaultRegistry, time.Nanosecond, *metricsPrefix, *graphiteAddr)
+	}
+
+	hostname, err := os.Hostname()
```

Comment:
Also need it if status port is set. I could do it doubly nested, but I think that would make the code unreadable.